### PR TITLE
English translation, HwcSwitch and HeatingSwitch made writable

### PR DESCRIPTION
**Pull request by Dinth**
This is my first github pull request so apologies if i made something incorrectly

### CODE REVIEW

**Comment by john30** [comment]
please also update the german file with the writeable HeatingSwitch

**Comment by Dinth** [comment]
Done. Just for the record the change was tested on Vaillant ecoTec 825 Plus.

File: ebusd-2.1.x/en/vaillant/bai.0010015600.inc [hunk 1]
```diff
@@ -6,165 +6,165 @@
...
-r,,PrimaryCircuitFlowrate,d.29 Wasserumlaufmenge,,,,"FB00",,,uin100,,l/min,value of flowrate,,,
-r,,TargetFanSpeed,d.33 Lüfter Solldrehzahl,,,,"2400",,,UIN,,1/min,Drehzahlsollwert des Lüfters,,,
-r,,FanSpeed,d.34 Lüfteristdrehzahl,,,,"8300",,,UIN,,1/min,Aktuelle Lüfterdrehzahl,,,
-r,,PositionValveSet,d.35 Position VUV,,,,"5400",,,UCH,,,Position des 3-Wege Ventil: 100=Warmwasser 0=Heizbetrieb 40=Mittenstellung,,,
-r,,FlowTemp,d.40 Vorlauftemperatur,,,,"1800",,,tempsensor,,,Vorlauftemperatur,,,
-r,,ReturnTemp,d.41 Rücklauftemperatur,,,,"9800",,,tempmirrorsensor,,,Rücklauftemperatur,,,
-r,,IonisationVoltageLevel,d.44 Spannungspegel Ionisationssignal,,,,"A400",,,SIN,10,,Ionisationsspannung: größer 80 = keine Flammekleiner 40 = gutes Flammensignal,,,
-r,,OutdoorstempSensor,d.47 Außentemperaturfühler,,,,"7600",,,tempsensor,,,Außentemperaturwert (ohne Korrekturwert) und Sensorstatus,,,
-r,,RemainingBoilerblocktime,d.67 Verbleibende Brennersperrzeit,,,,"3800",,,minutes0,,,Verbleibende Brennersperrzeit,,,
+r;wi,,PartloadHcKW,d.00 Heating part load,,,,”6C00",,,power,,,Heating part load,,,
```
Comment (john30) [inline ebusd-2.1.x/en/vaillant/bai.0010015600.inc]: the opening dquote character is wrong. this is the same for many lines

Comment (Dinth) [reply ebusd-2.1.x/en/vaillant/bai.0010015600.inc]: That should be fixed in the latest commit

File: ebusd-2.1.x/en/vaillant/bai.0010015600.inc [hunk 2]
```diff
@@ -6,165 +6,165 @@
...
-r,,OutdoorstempSensor,d.47 Außentemperaturfühler,,,,"7600",,,tempsensor,,,Außentemperaturwert (ohne Korrekturwert) und Sensorstatus,,,
-r,,RemainingBoilerblocktime,d.67 Verbleibende Brennersperrzeit,,,,"3800",,,minutes0,,,Verbleibende Brennersperrzeit,,,
+r;wi,,PartloadHcKW,d.00 Heating part load,,,,"6C00",,,power,,,Heating part load,,,
+r;wi,,WPPostrunTime,d.01 central heating overrun time,,,,"6400",,,minutes0,,,water pump overrun time for heating mode,,,
+r;wi,,BlockTimeHcMax,d.02 Max blocking time CH,,,,"2100",,,minutes0,,,Maximum burner anti cycling period at 20°C flow temperature set point,,,
+r,,HwcTemp,d.03 Temp DHW,,,,"1600",,,tempsensor,,,Hot water flow temperature,,,
+r,,StorageTemp,d.04 Storage temperature,,,,"1700",,,tempsensor,,,Current temperature for warm start sensor (combi boiler) / Current storage tank sensor (system boiler only),,,
+r,,FlowTempDesired,d.05 Flow/return set point,,,,"3900",,,temp,,,Flow temperature target value or return target value when return regulation is set,,,
+r,,StorageTempDesired,d.07 Storage temperature set point,,,,"0400",,,temp,,,Warm start temperature value (combi boiler only), Storage temperature target value (system boiler only),,,
```
Comment (john30) [inline ebusd-2.1.x/en/vaillant/bai.0010015600.inc]: the extra comma between the two comments needs to be removed

File: ebusd-2.1.x/en/vaillant/bai.0010015600.inc [hunk 3]
```diff
@@ -6,165 +6,165 @@
...
+r;wi,,PumpPowerDesired,d.14 Desired heating pump power,,,,A100,,,UCH,0=auto;1=53;2=60;3=70;4=85;5=100,%,PWM-Desired central heating pump power,,,
+r,,PumpPower,d.15 Current heating pump power,,,,7300,,,UCH,,,Current central heating pump power,,,
+r,,DCRoomthermostat,d.16 Room thermostat 24 V,,,,"0E00",,,onoff,,,xternal controls heat demand (Clamp 3-4),,,
+r,,HwcDemand,d.22 DHW demand,,,,"5800",,,yesno,,,Domestic hot water demand,,,
+r;wi,,HeatingSwitch,d.23 Heating on/off switch,,,,"F203",,,onoff,,,Central heating on/off switch,,,
+#r,,StoragereleaseClock,d.25 DHW demand enabled,,,,"4704",,,yesno,,,hot water release (tank storage) via eBus Control,,,
+r,,PrimaryCircuitFlowrate,d.29 Primary circuit flow rate,,,,"FB00",,,uin100,,l/min,Primary circuit water flow rate,,,
+r,,TargetFanSpeed,d.33 Target fan speed,,,,"2400",,,UIN,,1/min,Fan speed setpoint,,,
+r,,FanSpeed,d.34 Current fan speed,,,,"8300",,,UIN,,1/min,Current fan speed,,,
+r,,PositionValveSet,d.35 Position TWV,,,,"5400",,,UCH,,,Position of diverter valve; 100 = DHW, 0 = heating, 40 = middle position,,,
```
Comment (john30) [inline ebusd-2.1.x/en/vaillant/bai.0010015600.inc]: the extra commas between the values in the comment needs to be removed

File: ebusd-2.1.x/en/vaillant/bai.0010015600.inc [hunk 4]
```diff
@@ -6,165 +6,165 @@
...
-r,,DisplayMode,DisplayMode,,,,"DA00",,,UCH,,,Display mode of the aplliance,,,
-r,,Gasvalve3UC,Gasventil 3,,,,"DB00",,,UCH,240=off;15=on,,Gasventil Schaltsignal (vom Prozessor),,,
-r,,InitialisationEEPROM,InitialisationEEPROM,,,,"DC00",,,yesno,,,EEPROM Initialisierung (für die Produktion),,,
-r,,TimerInputHc,Eingang Schaltuhr,,,,"DE00",,,onoff,,,timer input (block heatdemand),,,
-r,,FanMinSpeedOperation,Min. Lüfterdrehzahl,,,,"DF00",,,UIN,,1/min,Lüfter Minimaldrehzahl,,,
-r,,FanMaxSpeedOperation,Max. Lüfterdrehzahl,,,,"E000",,,UIN,,1/min,Lüfter Maximaldrehzahl,,,
-r,,ExternGasvalve,Ext. Gasventil,,,,"E400",,,onoff,,,Externes Magnetventil,,,
-r,,DCFTimeDate,DCF Datum/Uhrzeit,,,,"E500",,,VTI;HDA,,,DCF Time / date stamp struct { uchar sec; uchar min; uchar std; uchar tag; uchar mon; uchar wtg; uchar jah; } Tdcf77_time;
-r,,HwcSwitch,Wasserschalter,,,,"F303",,,onoff,,,WW aus/ein,,,
+r,,TemplimiterWithNTC,SD_STL_with_NTC,,,,"D200",,,UCH,240=no;15=yes,,Temperature limiter (1 = NTC, 0 = switching contact),,,
```
Comment (john30) [inline ebusd-2.1.x/en/vaillant/bai.0010015600.inc]: the extra comma between the values in the comment need to be removed

**Comment by Dinth** [comment]
Fixed

**Pull request merged**